repo2docker/buildpacks/nix/nix-shell-wrapper

```diff
@@ -8,20 +8,27 @@ _term() {
 
 trap _term SIGTERM
 
+# find binder sub-directory (if present)
+for dir in "./.binder" "./binder"; do
+  if [ -e $dir ]; then
+    binder_dir=$dir
+  fi
+done
+
 # if there is a binder/ sub-directory it takes precedence
 # files outside it are ignored
-if [ -e ./binder ]; then
-  nixpath="./binder/default.nix";
-  if [ -f ./binder/start ]; then
-    chmod u+x ./binder/start
+if [ -z $binder_dir ]; then
+  nixpath="$binder_dir/default.nix";
```

Comment:
From the failing nix tests it looks like it ends up setting `nixpath` to `/default.nix` and then erroring out. Not quite sure why it happens though :-/